spacy/pipeline/spancat_exclusive.py

```diff
@@ -266,23 +266,26 @@ def _make_span_group(
     ) -> SpanGroup:
         scores = self.model.ops.to_numpy(scores)
         indices = self.model.ops.to_numpy(indices)
-        predicted = scores.argmax(axis=1)
-
-        # Remove samples where the negative label is the argmax
-        positive = numpy.where(predicted != self._negative_label)[0]
-        predicted = predicted[positive]
-        indices = indices[positive]
-
-        # Sort spans according to argmax probability
-        if not allow_overlap:
-            # Get the probabilities
-            argmax_probs = numpy.take_along_axis(
-                scores[positive], numpy.expand_dims(predicted, 1), axis=1
-            )
-            argmax_probs = argmax_probs.squeeze()
-            sort_idx = (argmax_probs * -1).argsort()
-            predicted = predicted[sort_idx]
-            indices = indices[sort_idx]
+        if scores.size != 0:
```

Comment:
I'm not sure if this is the most elegant solution. The goal is that `_make_span_group` should return an empty `SpanGroup` when there zero suggestions. The problem happens when the `scores` variable is empty, and we cannot get all the other variables from that. I enclosed them in an if statement so that another branch happens.

Reply:
maybe its more elegant (?) to have this on the very top:

```python
if scores.size == 0:
    return SpanGroup(doc, name=self.key)
scores = self.model.ops.to_numpy(scores)
indices = self.model.ops.to_numpy(indices)
...
```

Reply:
Ah that's nice! Ok will update in a few